test/BlazorSignalR.Test.RazorComponents/Controllers/TokenController.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+
+namespace BlazorSignalR.Test.RazorComponents.Controllers
+{
+    [Route("generatetoken")]
+    public class TokenController : Controller
+    {
+        [HttpGet]
+        public string GenerateToken()
+        {
+            var claims = new[] { new Claim(ClaimTypes.NameIdentifier, this.Request.Query["user"]) };
```

Comment:
The ```this``` is unneeded.

Reply:
This is true, but I just copied the code from the existing blazor server sample project. Maybe we should add an .editorconfig file to help with the code styling independent of the individual VS settings. We could copy the one from [my project](https://github.com/AI4E/AI4E/blob/master/.editorconfig) and modify it to your needs.

Reply:
Generally the project just follows the Resharper defaults.

Reply:
That makes sense, but many people do not have Resharper installed. I do not because VS is that slow with very large projects then, that it is nearly unusable.